pkg/reconciler/openshift/tektonaddon/tektonaddon.go

```diff
@@ -369,26 +354,34 @@ func (r *Reconciler) ensureClusterTasks(ctx context.Context, ta *v1alpha1.Tekton
 // and if installer set which already exist is of older version then it deletes and return false to create a new
 // installer set
 func checkIfInstallerSetExist(ctx context.Context, oc clientset.Interface, relVersion string,
```

Comment:
as this is more generalised now, can we move this to installer Set pkg
so that other component can use the same?
something like `installerSet.GetByLabel()`

Reply:
separate pr sound fine. 
could you create and issue (or jira) to track this.